examples/binding_affinity_prediction/utils.py

```diff
@@ -46,20 +64,36 @@ def load_dataset(args):
     """
     assert args['dataset'] in ['PDBBind'], 'Unexpected dataset {}'.format(args['dataset'])
     if args['dataset'] == 'PDBBind':
-        dataset = PDBBind(subset=args['subset'],
+        if args['model'] == 'PotentialNet': 
+            from functools import partial
+            from dgllife.utils import potentialNet_graph_construction_featurization
+            dataset = PDBBind(subset=args['subset'], pdb_version=args['version'], 
+                    remove_coreset_from_refinedset=args['remove_coreset_from_refinedset'],
+                    load_binding_pocket=args['load_binding_pocket'],
+                    construct_graph_and_featurize = partial(potentialNet_graph_construction_featurization, 
+                    distance_bins=args['distance_bins'],
+                    max_num_neighbors=args['max_num_neighbors'])
```

Comment:
Fix the indent for L74 - 75, i.e. the first letter in "distance_bins" and "max_num_neighbors" should align with the first letter in "potentialNet".